packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -853,6 +853,19 @@ const ArrayMixin = Mixin.create(Enumerable, {
     object that will be set as "this" on the context. This is a good way
     to give your iterator function access to the current object.
 
+    Example Usage:
+
+    ```javascript
+    const food = [
+      { food: 'apple', isFruit: true},
+      { food: 'bread', isFruit: false },
+      { food: 'banana', isFruit: true }
+    ];
+    const nonFruits = food.reject(function(thing) {
+      return thing.isFruit;
+    }); // [{food: 'beans', isFruit: false}]
```

Comment:
This should be `bread` not `beans` right?

Reply:
I suppose to be consistent with my other pull requests, yes. 🍞

Reply:
Ah sorry, I see my mistake now. I will fix it.

Reply:
Fixed here https://github.com/emberjs/ember.js/pull/18503/commits/ed2fe07eac1f94efd5e849233ceeeba95776c9b3